src/main/java/org/opentripplanner/index/IndexGraphQLSchema.java

```diff
@@ -1701,7 +1701,9 @@ public IndexGraphQLSchema(GraphIndex index) {
                     if (environment.getArgument("name") != null) {
                         stream = stream
                             .filter(route -> route.getShortName() != null)
-                            .filter(route -> route.getShortName().startsWith(environment.getArgument("name")));
+                            .filter(route -> route.getShortName().toLowerCase().startsWith(
+                                    environment.getArgument("name").toLowerCase())
```

Comment:
`environment.getArgument("name")` needs type check and casting in order to be able to use toLowerCase